mp/src/game/shared/momentum/mom_rocket.cpp

```diff
@@ -195,8 +206,39 @@ void CMomRocket::Explode(trace_t *pTrace, CBaseEntity *pOther)
     float flDamage = GetDamage();
     float flRadius = GetRadius();
 
-    // Create explosion effect with no damage
-    ExplosionCreate(vecOrigin, GetAbsAngles(), pOwner, flDamage, flRadius, false);
+    static ConVarRef mom_rj_sounds("mom_rj_sounds");
+    CPVSFilter filter(vecOrigin);
+    int EntIndex = 0;
+    EntIndex = pOther->entindex();
```

Comment:
Try to not use `UpperCamelCase` when declaring variables here, it's confusing to read and could potentially interfere with class/enum names. Use `lowerCamelCase` instead, and join declaration and assignment:
```
int iEntIndex = pOther->entindex();
```